tests/test_github.py

```diff
@@ -0,0 +1,746 @@
+                "url": "https://api.github.com/repos/zhquan_example/repo/issues/278",
+                "body": "body",
+                "user_data": {
+                    "type": "User",
+                    "organizations_url": "https://api.github.com/users/zhquan_example/orgs",
+                    "repos_url": "https://api.github.com/users/zhquan_example/repos",
+                    "organizations": [
+                        {
+                            "login": "Orgs_1",
+                            "members_url": "https://api.github.com/orgs/Orgs_1/members{/member}",
+                            "avatar_url": "",
+                            "events_url": "https://api.github.com/orgs/Orgs_1/events",
+                            "hooks_url": "https://api.github.com/orgs/Orgs_1/hooks",
+                            "id": 1,
+                            "public_members_url": "https://api.github.com/orgs/Orgs_1/public_members{/member}",
+                            "repos_url": "https://api.github.com/orgs/Orgs_1/repos",
+                            "issues_url": "https://api.github.com/orgs/Orgs_1/issues",
+                            "url": "https://api.github.com/orgs/Orgs_1",
+                            "description": None
+                        },
+                        {
+                            "login": "Orgs_2",
+                            "members_url": "https://api.github.com/orgs/Orgs_2/members{/member}",
+                            "avatar_url": "",
+                            "events_url": "https://api.github.com/orgs/Orgs_2/events",
+                            "hooks_url": "https://api.github.com/orgs/Orgs_2/hooks",
+                            "id": 2,
+                            "public_members_url": "https://api.github.com/orgs/Orgs_2/public_members{/member}",
+                            "repos_url": "https://api.github.com/orgs/Orgs_2/repos",
+                            "issues_url": "https://api.github.com/orgs/Orgs_2/issues",
+                            "url": "https://api.github.com/orgs/Orgs_2",
+                            "description": None
+                        }
+                    ],
+                    "name": "zhquan_example",
+                    "url": "https://api.github.com/users/zhquan_example",
+                    "created_at": "2016-01-01T00:00:00Z",
+                    "updated_at": "2016-01-01T01:00:00Z",
+                    "login": "zhquan_example",
+                    "id": 1
+                },
+                "created_at": "2016-01-22T09:54:47Z",
+                "closed_at": "2016-03-15T15:09:29Z",
+                "id": 2
+            },
+            "perceval_version": "0.1.0",
+            "updated_on": 1458054569.0,
+            "origin": "https://github.com/zhquan_example/repo",
+            "uuid": "4236619ac2073491640f1698b5c4e169895aaf69",
+            "backend_version": "0.1.0"
+        }
+
+        for key in expected:
```

Comment:
Same comment that above but you also have to check the contents of the two issues, not just only one.